kiwi/bootloader/config/grub2.py

```diff
@@ -64,8 +64,8 @@ def post_init(self, custom_args):
         self.firmware = FirmWare(
             self.xml_state
         )
-
-        self.hybrid_boot = True
+        if arch == 'x86_64':
+            self.hybrid_boot = True
```

Comment:
imho we shouldn't do that. The condition will break hybrid_boot for ix86 and I think there shouldn't be such a condition at all. hybrid_boot just causes a grub.cfg which uses $linux and $initrd grub variables to check for the grub_cpu variable and decide for the right grub command. That should also work on ppc. You might want to change kiwi/bootloader/template/grub2 and there the self.header_hybrid setup